vdr/vdr.go

```diff
@@ -147,7 +147,7 @@ func (r VDR) Update(id did.DID, current hash.SHA256Hash, next did.Document, _ *t
 		Hash:             &current,
 		AllowDeactivated: true,
 	}
-	currentDIDDocument, _, err := r.store.Resolve(id, resolverMetadata)
+	currentDIDDocument, currentMeta, err := r.store.Resolve(id, resolverMetadata)
```

Comment:
in context of the `current` parameter, what if there are multiple? Just pick one?

Reply:
the current hash refers to the hash of the current (merged) doc. There's no multiple, just one